packages/cactus-plugin-ledger-connector-besu/src/test/typescript/integration/plugin-ledger-connector-besu/deploy-contract/get-record-locator.test.ts

```diff
@@ -0,0 +1,302 @@
+      methodName: "setName",
+      params: [newName],
+      signingCredential: {
+        ethAccount: testEthAccount.address,
+        secret: testEthAccount.privateKey,
+        type: Web3SigningCredentialType.PrivateKeyHex,
+      },
+      nonce: 1,
+    });
+    t2.ok(setNameOut, "setName() invocation #1 output is truthy OK");
+
+    try {
+      const setNameOutInvalid = await connector.invokeContract({
+        contractName: HelloWorldContractJson.contractName,
+        contractAbi: HelloWorldContractJson.abi,
+        contractAddress,
+        invocationType: EthContractInvocationType.Send,
+        methodName: "setName",
+        params: [newName],
+        gas: 1000000,
+        signingCredential: {
+          ethAccount: testEthAccount.address,
+          secret: testEthAccount.privateKey,
+          type: Web3SigningCredentialType.PrivateKeyHex,
+        },
+        nonce: 1,
+      });
+      t2.ifError(setNameOutInvalid);
+    } catch (error) {
+      t2.notStrictEqual(
+        error,
+        "Nonce too low",
+        "setName() invocation with invalid nonce",
+      );
+    }
+    const req: InvokeContractV1Request = {
+      contractName: HelloWorldContractJson.contractName,
+      contractAbi: HelloWorldContractJson.abi,
+      contractAddress,
+      invocationType: EthContractInvocationType.Call,
+      methodName: "getName",
+      params: [],
+      gas: 1000000,
+      signingCredential: {
+        ethAccount: testEthAccount.address,
+        secret: testEthAccount.privateKey,
+        type: Web3SigningCredentialType.PrivateKeyHex,
+      },
+    };
+    const { callOutput: getNameOut } = await connector.getBesuRecord({
+      invokeCall: req,
+    });
+    t2.equal(getNameOut, newName, `getName() output reflects the update OK`);
+
+    const response = await connector.invokeContract({
+      contractName: HelloWorldContractJson.contractName,
+      contractAbi: HelloWorldContractJson.abi,
+      contractAddress,
+      invocationType: EthContractInvocationType.Send,
+      methodName: "deposit",
+      params: [],
+      gas: 1000000,
+      signingCredential: {
+        ethAccount: testEthAccount.address,
+        secret: testEthAccount.privateKey,
+        type: Web3SigningCredentialType.PrivateKeyHex,
+      },
+      value: 10,
+    });
+    t2.ok(response, "deposit() payable invocation output is truthy OK");
+
+    const req2: InvokeContractV1Request = {
+      contractName: HelloWorldContractJson.contractName,
+      contractAbi: HelloWorldContractJson.abi,
+      contractAddress,
+      invocationType: EthContractInvocationType.Call,
+      methodName: "getNameByIndex",
+      params: [0],
+      gas: 1000000,
+      signingCredential: {
+        ethAccount: testEthAccount.address,
+        secret: testEthAccount.privateKey,
+        type: Web3SigningCredentialType.PrivateKeyHex,
+      },
+    };
+
+    const { callOutput } = await connector.getBesuRecord({ invokeCall: req2 });
```

Comment:
@AzaharaC Please use the API client instead of the connector for at least one of these tests to ensure that the endpoint class is also covered by the tests. Apologies for not noticing this earlier!